### PR TITLE
Shell arg fix

### LINKED ISSUE #374: withokta saving to correct aws profile
withokta script is no longer writing the inputted profile new for the ~/.aws/credentials file block.

1. okta-aws default sts get-caller-identity
2. Select profile (e.g. Account 1234 - role-sandbox-developer)
3. Check ~/.aws/credentials file
4. Shows authenticated block as [role-sandbox-developer_1234]

**Expected behavior**
Should be writing to ~/.aws/credentials under [default] block (or other inputted name)

**Workaround**
Reverted back to old withokta script worked
https://github.com/oktadev/okta-aws-cli-assume-role/commit/63b880957e0d64cd3387e1fbf16e6aa7e808db65#diff-0bb30e60c9060eae4f3b622c188c4ed97bc18c04242404cd8a9a4098961082e4

**Additional context**
Java - 12.0.1
okta-aws-cli.jar - 2.0.5
aws-cli - 2.2.23
Python - 3.8.8
MacOS - 11.4 Darwin/20.5.0

### PR DESCRIPTION
**Pull request by dangeReis**
Problem Statement
Updating the withOkta script to not use the profile broke the okta-aws and okta-sls commands.

Solution
Set the profile in the respective commands.  Also remove the profile from the arguments being passed in to aws cli (or sls cli).  Additionally this stores the appropriate profile name in the .aws/credentials file.  

This should fix #374 #375 #376

### CODE REVIEW

**Comment by dangeReis** [comment]
replaces #378 

@hcourse-nydig @DavidTanner please confirm that this looks good.  I basically took both of your solutions and combined them.  Also sorry, not sure if there's a way to modify an existing PR, so created a new one.

File: bin/install.sh [hunk 1]
```diff
@@ -111,12 +111,14 @@ fishFunctionsDir="${PREFIX}/fish_functions"
...
-    withokta "aws --profile $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "aws --profile $argv[1]" $argv[2..-1]
 end
 EOF
 cat <<'EOF' >"${fishFunctionsDir}/okta-sls.fish"
 function okta-sls
-    withokta "sls --stage $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "sls --stage $argv[1]" $argv[2..-1]
```
Comment (DavidTanner) [inline bin/install.sh:121]: I'm not sure what this line will do.  From what I learned of fish, line 120 is the one to keep, and remove `OKTA_PROFILE="$1"` from this line.

File: bin/install.sh [hunk 2]
```diff
@@ -111,12 +111,14 @@ fishFunctionsDir="${PREFIX}/fish_functions"
 mkdir -p "${fishFunctionsDir}"
 cat <<'EOF' >"${fishFunctionsDir}/okta-aws.fish"
 function okta-aws
-    withokta "aws --profile $argv[1]" $argv
+    set -lx OKTA_PROFILE "$argv[1]"
+    OKTA_PROFILE="$1" withokta "aws --profile $argv[1]" $argv[2..-1]
```
Comment (DavidTanner) [inline bin/install.sh:115]: Same as below.  Doesn't work the same in fish as bash.

**Review by DavidTanner — CHANGES_REQUESTED** [review]

File: bin/install.sh (hunk 2 above)
Comment (dangeReis) [reply bin/install.sh]: fixed

File: bin/install.sh (hunk 1 above)
Comment (dangeReis) [reply bin/install.sh]: fixed

**Review by DavidTanner — APPROVED** [review]
LGTM

**Comment by dangeReis** [comment]
@mraible what stands in the way of getting this merged in?

**Pull request merged**